### ISSUE TITLE
@file or @overview tags ignore

### ISSUE THREAD

**Issue by madtrick** [jsdoc2md/jsdoc-to-markdown#84]
### Summary

jsdoc tags `@file` and its synonims `@overview` and `@fileoverview` seem to be ignored by `jsdoc-to-markdown`

#### Steps to reproduce
Create a file with the following contents:

```js
'use strict';

/**
 * @file A sample file
 */

/**
 * @function foo
 */
function foo () {
  return 'bar';
}
```

Run `jsdoc2md`:

```bash
$ jsdoc2md file.js
<a name="foo"></a>

## foo()
**Kind**: global function
```

As you can see the contents for the `@file` tag are not included in the generated markdown.

#### Other information

Using version `1.3.6.` of `jsdoc-to-markdown`

**Comment by 75lb** [comment]
i don't think this tag gets much usage in jsdoc2md as the information it contains has no impact on a module's API (which is what the tool is intended to document).. But will have a look, thanks.

**Comment by timothyswt** [comment]
I there a workaround for this? I need to be able to document files as well.

**Comment by timothyswt** [comment]
I figured out a workaround. just use the "@name" tag instead and just give your file a name without an extension. then it will pick up the comments regardless. just don't use "@file" or synonyms or it will still ignore it even when using "@name"

**Comment by cameronelliott** [comment]
> I figured out a workaround. just use the "@name" tag instead and just give your file a name without an extension. then it will pick up the comments regardless. just don't use "@file" or synonyms or it will still ignore it even when using "@name"

This doesn't really seem to be a substitute for @file , it can be used to document a function defined outside the source file, though.

**Comment by dmnsgn** [comment]
I'd be interested in this feature too 👍
